shakenfist/tasks.py

```diff
@@ -41,10 +41,12 @@ def json_dump(self):
 # Instance Tasks
 #
 class InstanceTask(QueueTask):
-    def __init__(self, instance_uuid, network=None):
+    def __init__(self, instance_uuid, network=[]):
```

Comment:
You can't actually do this in python. You must use None in the method signature and then turn that into an empty list below. https://stackoverflow.com/questions/366422/what-is-the-pythonic-way-to-avoid-default-parameters-that-are-empty-lists has discussion of why, but it comes down to when the interpreter executes the method signature (once on initial parse).

Reply:
Very good point.